### ISSUE TITLE
[REVIEW]: simode: R Package for Statistical Inference of Ordinary Differential Equations using Separable Integral-Matching

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#1850]
**Submitting author:** @ramiyaari (<a href="http://orcid.org/0000-0002-8808-8937">Rami Yaari</a>)
**Repository:** <a href="https://github.com/ramiyaari/simode/" target ="_blank">https://github.com/ramiyaari/simode/</a>
**Version:** v1.1.8
**Editor:** @mjsottile
**Reviewer:** @cscherrer, @jgoldfar, @osorensen
**Archive:** <a href="https://doi.org/10.5281/zenodo.3576265" target="_blank">10.5281/zenodo.3576265</a>

## Status

[![status](https://joss.theoj.org/papers/645cf34d694b5976179507afb8b7d61c/status.svg)](https://joss.theoj.org/papers/645cf34d694b5976179507afb8b7d61c)

Status badge code:

```
HTML: <a href="https://joss.theoj.org/papers/645cf34d694b5976179507afb8b7d61c"><img src="https://joss.theoj.org/papers/645cf34d694b5976179507afb8b7d61c/status.svg"></a>
Markdown: [![status](https://joss.theoj.org/papers/645cf34d694b5976179507afb8b7d61c/status.svg)](https://joss.theoj.org/papers/645cf34d694b5976179507afb8b7d61c)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/ramiyaari/simode/" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) by leaving comments in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

## Reviewer instructions & questions

@cscherrer & @jgoldfar & @osorensen, please carry out your review in this issue by updating the checklist below. If you cannot edit the checklist please:
1. Make sure you're logged in to your GitHub account
2. Be sure to accept the invite at this URL: https://github.com/openjournals/joss-reviews/invitations

The reviewer guidelines are available here: https://joss.readthedocs.io/en/latest/reviewer_guidelines.html. Any questions/concerns please let @mjsottile know.

✨ **Please try and complete your review in the next two weeks** ✨ 

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/ramiyaari/simode/">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@ramiyaari) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/ramiyaari/simode/">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@ramiyaari) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

### Conflict of interest

- [x] I confirm that I have read the [JOSS conflict of interest (COI) policy](https://github.com/openjournals/joss/blob/master/COI.md) and that: I have no COIs with reviewing this work or that any perceived COIs have been waived by JOSS for the purpose of this review.

### Code of Conduct

- [x] I confirm that I read and will adhere to the [JOSS code of conduct](https://joss.theoj.org/about#code_of_conduct).

### General checks

- [x] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/ramiyaari/simode/">repository url</a>?
- [x] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [x] **Contribution and authorship:** Has the submitting author (@ramiyaari) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?

### Functionality

- [x] **Installation:** Does installation proceed as outlined in the documentation?
- [x] **Functionality:** Have the functional claims of the software been confirmed?
- [x] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)

### Documentation

- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [x] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [x] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g., API method documentation)?
- [x] **Automated tests:** Are there automated tests or manual steps described so that the functionality of the software can be verified?
- [x] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [x] **Summary:** Has a clear description of the high-level functionality and purpose of the software for a diverse, non-specialist audience been provided?
- [x] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [x] **State of the field:** Do the authors describe how this software compares to other commonly-used packages?
- [x] **Quality of writing:** Is the paper well written (i.e., it does not require editing for structure, language, or writing quality)?
- [x] **References:** Is the list of references complete, and is everything cited appropriately that should be cited (e.g., papers, datasets, software)? Do references in the text use the proper [citation syntax]( https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)?

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks. @cscherrer, @jgoldfar, @osorensen it looks like you're currently assigned to review this paper :tada:.

**:star: Important :star:**

If you haven't already, you should seriously consider unsubscribing from GitHub notifications for this (https://github.com/openjournals/joss-reviews) repository. As a reviewer, you're probably currently watching this repository which means for GitHub's default behaviour you will receive notifications (emails) for all reviews 😿

To fix this do the following two things:

1. Set yourself as 'Not watching' https://github.com/openjournals/joss-reviews:

![watching](https://cloud.githubusercontent.com/assets/4483/20250593/64d7ce48-a9de-11e6-9225-d3dfb3e48e68.png)

2. You may also like to change your default settings for this watching repositories in your GitHub profile here: https://github.com/settings/notifications

![notifications](https://cloud.githubusercontent.com/assets/4483/20250604/79eb3144-a9de-11e6-9e51-0fe940966689.png)


For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01850/joss.01850/10.21105.joss.01850.pdf)

**Comment by cscherrer** [comment]
https://github.com/ramiyaari/simode/issues/5

**Comment by osorensen** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01850/joss.01850/10.21105.joss.01850.pdf)

**Comment by ramiyaari** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01850/joss.01850/10.21105.joss.01850.pdf)

**Comment by mjsottile** [comment]
It looks like we only have a couple boxes to check in the review checklists above.  @cscherrer  and @jgoldfar - can you take a look at the remaining items on your checklists to see if they have been addressed to your satisfaction?

**Comment by jgoldfar** [comment]
Done

**Comment by mjsottile** [comment]
@ramiyaari it looks like the review checklists are complete.  I will look everything over in a few days when I return from travel and see what the next steps are.

**Comment by mjsottile** [comment]
Can the reviewers @cscherrer, @jgoldfar, @osorensen confirm their recommendation to accept the submission by responding in this thread?

Also, @ramiyaari - I see that there is a pull request still waiting for a documentation correction.  Can you address that and close the open pull request?

**Comment by osorensen** [comment]
@mjsottile , I confirm my recommendation.

**Comment by ramiyaari** [comment]
Thanks @mjsottile . I've merged the pull request.

**Comment by jgoldfar** [comment]
@mjsottile I confirm my recommendation

**Comment by cscherrer** [comment]
Sorry I had missed this. Yes, I confirm my recommendation

**Comment by mjsottile** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01850/joss.01850/10.21105.joss.01850.pdf)

**Comment by mjsottile** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.18637/jss.v033.i09 is OK
- 10.1177/0962280217746443 is OK
- 10.1214/15-EJS1053 is OK

MISSING DOIs

- https://doi.org/10.1371/journal.pcbi.1004012 may be missing for title: 150 years of the mass action law
- https://doi.org/10.1098/rsif.2016.0525 may be missing for title: Modelling and parameter inference of predator–prey dynamics in heterogeneous environments using the direct integral approach
- https://doi.org/10.1016/j.spl.2017.08.013 may be missing for title: Consistency of direct integral estimator for partially observed systems of ordinary differential equations
- https://doi.org/10.1111/stan.12124 may be missing for title: Application of one-step method to parameter estimation in ODE models
- https://doi.org/10.18637/jss.v075.i02 may be missing for title: CollocInfer: Collocation Inference in Differential Equation Models
- https://doi.org/10.1007/s11222-014-9486-9 may be missing for title: Time-course window estimator for ordinary differential equations linear in the parameters
- https://doi.org/10.1111/biom.12348 may be missing for title: A model-based initial guess for estimating parameters in systems of ordinary differential equations
- https://doi.org/10.1098/rsif.2010.0412 may be missing for title: Parameterizing state–space models for infectious disease dynamics by generalized profiling: measles in Ontario
- https://doi.org/10.1016/j.jmva.2008.03.005 may be missing for title: Statistical properties of the Hough transform estimator in the presence of measurement errors
- https://doi.org/10.1111/j.1541-0420.2008.01172.x may be missing for title: Forcing function diagnostics for nonlinear dynamics
- https://doi.org/10.1093/bioinformatics/bth140 may be missing for title: Decoupling dynamical systems for pathway identification from metabolic profiles

INVALID DOIs

- None
```

**Comment by mjsottile** [comment]
Only a few steps left to finalize acceptance @ramiyaari 

1. Please review the pull request I just submitted.  I did a final pass over the paper and found a small number of minor changes to the text.  
2. Please add the missing DOIs to the references that lack them. See the message from whedon above.
3. Once you have completed this, the last step is to make a tagged release and archive.  Please comment in this thread with the version number and archive DOI.

Once those three steps are completed, I will send the submission off to the JOSS EICS for final processing.

**Comment by ramiyaari** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01850/joss.01850/10.21105.joss.01850.pdf)

**Comment by ramiyaari** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01850/joss.01850/10.21105.joss.01850.pdf)

**Comment by ramiyaari** [comment]
@whedon check references

**Comment by whedon** [comment]
```
Attempting to check references...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.18637/jss.v033.i09 is OK
- 10.1155/2013/897658 is OK
- 10.1371/journal.pcbi.1004012 is OK
- 10.1098/rsif.2016.0525 is OK
- 10.1016/j.spl.2017.08.013 is OK
- 10.1177/0962280217746443 is OK
- 10.1111/stan.12124 is OK
- 10.18637/jss.v075.i02 is OK
- 10.1007/s11222-014-9486-9 is OK
- 10.1214/15-EJS1053 is OK
- 10.1111/biom.12348 is OK
- 10.1098/rsif.2010.0412 is OK
- 10.1016/j.jmva.2008.03.005 is OK
- 10.1111/j.1541-0420.2008.01172.x is OK
- 10.1093/bioinformatics/bth140 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by ramiyaari** [comment]
Hi @mjsottile , I've completed the first two tasks in your list. Regarding the third, if I understand what you've asked for then the current version of the software in CRAN is 1.1.8, and the doi of the joss paper is 10.21105/joss.01850.
Thanks

**Comment by danielskatz** [comment]
For the second part, JOSS needs the software to be archived in an archival repository, such as Zenodo, figshare, or an institutional repository.  Once this is done, that repository should provide a DOI to the archive, which is what @mjsottile needs.  For software in GitHub, the easiest path is usually to use to the [Zenodo-GitHub linkage](https://guides.github.com/activities/citable-code/), being sure to manually change the metadata in Zenodo so that the title and authors of the archive match the those of the JOSS paper.

**Comment by ramiyaari** [comment]
Thanks, @danielskatz and @mjsottile.
I followed your instructions and created the archive using Zenodo.
The doi to the archive is 10.5281/zenodo.3576265
Target URL is https://zenodo.org/badge/latestdoi/142470608

**Comment by mjsottile** [comment]
@whedon set 10.5281/zenodo.3576265 as archive

**Comment by whedon** [comment]
OK. <a href="https://doi.org/10.5281/zenodo.3576265" target="_blank">10.5281/zenodo.3576265</a> is the archive.

**Comment by mjsottile** [comment]
@whedon set v1.1.8 as version

**Comment by whedon** [comment]
OK. v1.1.8 is the version.

**Comment by mjsottile** [comment]
Thanks @ramiyaari - I believe we're ready for final processing.

**Comment by mjsottile** [comment]
:wave: @openjournals/joss-eics This paper has completed the review process and I believe it's ready for final processing.  Let me know if there is anything additional that I should do.

**Comment by ramiyaari** [comment]
Thank you @mjsottile

**Comment by labarba** [comment]
@ramiyaari — I'm the associate EiC on rotation, and will process your paper to publication. We ask authors to edit the metadata (in this case, the title) of the Zenodo deposit so it matches the JOSS paper. Could you make that change?

**Comment by labarba** [comment]
Looking at the paper, it does look rather long for the typical JOSS paper. In fact, the [JOSS documentation](https://joss.readthedocs.io/en/latest/submitting.html) stipulates that papers should be _short_: "The paper should be between 250-1000 words." And also,

> a “full length” paper is not permitted, and software documentation such as API (Application Programming Interface) functionality should not be in the paper and instead should be outlined in the software documentation.

Do you think the detailed example should rather be in the documentation, and simply be mentioned here with a link?

**Comment by ramiyaari** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
PDF failed to compile for issue #1850 with the following error: 

 Can't find any papers to compile :-(

**Comment by ramiyaari** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01850/joss.01850/10.21105.joss.01850.pdf)

**Comment by ramiyaari** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
```
Attempting PDF compilation. Reticulating splines etc...
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.01850/joss.01850/10.21105.joss.01850.pdf)

**Comment by ramiyaari** [comment]
Hi @labarba, I've fixed the title of the Zenodo to match that of the JOSS paper.
I've also edited the paper and reduced the space taken by the example by two pages.
If that is still too long than I will remove the example completely,
but I was looking for example at this paper (https://joss.theoj.org/papers/10.21105/joss.01601)
and saw it is similar in length to what we have now.

**Comment by labarba** [comment]
@whedon accept

**Comment by whedon** [comment]
```
Attempting dry run of processing paper acceptance...
```

**Comment by whedon** [comment]
```Reference check summary:

OK DOIs

- 10.18637/jss.v033.i09 is OK
- 10.1155/2013/897658 is OK
- 10.1371/journal.pcbi.1004012 is OK
- 10.1098/rsif.2016.0525 is OK
- 10.1016/j.spl.2017.08.013 is OK
- 10.1177/0962280217746443 is OK
- 10.1111/stan.12124 is OK
- 10.18637/jss.v075.i02 is OK
- 10.1007/s11222-014-9486-9 is OK
- 10.1214/15-EJS1053 is OK
- 10.1111/biom.12348 is OK
- 10.1098/rsif.2010.0412 is OK
- 10.1016/j.jmva.2008.03.005 is OK
- 10.1111/j.1541-0420.2008.01172.x is OK
- 10.1093/bioinformatics/bth140 is OK

MISSING DOIs

- None

INVALID DOIs

- None
```

**Comment by whedon** [comment]
Check final proof :point_right: https://github.com/openjournals/joss-papers/pull/1198

If the paper PDF and Crossref deposit XML look good in https://github.com/openjournals/joss-papers/pull/1198, then you can now move forward with accepting the submission by compiling again with the flag `deposit=true` e.g.
 ```
@whedon accept deposit=true
```

**Comment by labarba** [comment]
@whedon accept deposit=true

**Comment by whedon** [comment]
```
Doing it live! Attempting automated processing of paper acceptance...
```

**Comment by whedon** [comment]
🐦🐦🐦 👉 [Tweet for this paper](https://twitter.com/JOSS_TheOJ/status/1208890612596051968) 👈 🐦🐦🐦

**Comment by whedon** [comment]
🚨🚨🚨 **THIS IS NOT A DRILL, YOU HAVE JUST ACCEPTED A PAPER INTO JOSS!** 🚨🚨🚨

 Here's what you must now do:

0. Check final PDF and Crossref metadata that was deposited :point_right: https://github.com/openjournals/joss-papers/pull/1199
1. Wait a couple of minutes to verify that the paper DOI resolves [https://doi.org/10.21105/joss.01850](https://doi.org/10.21105/joss.01850)
2. If everything looks good, then close this review issue.
3. Party like you just published a paper! 🎉🌈🦄💃👻🤘

 Any issues? notify your editorial technical team...

**Comment by labarba** [comment]
Congratulations, @ramiyaari, your JOSS paper is published! 🚀 

Huge thanks to our editor: @mjsottile, and the reviewers: @cscherrer, @jgoldfar, @osorensen — we are so grateful for your contributions 🙏

**Comment by whedon** [comment]
:tada::tada::tada: Congratulations on your paper acceptance! :tada::tada::tada:

If you would like to include a link to your paper from your README use the following code snippets:

```
Markdown:
[![DOI](https://joss.theoj.org/papers/10.21105/joss.01850/status.svg)](https://doi.org/10.21105/joss.01850)

HTML:
<a style="border-width:0" href="https://doi.org/10.21105/joss.01850">
  <img src="https://joss.theoj.org/papers/10.21105/joss.01850/status.svg" alt="DOI badge" >
</a>

reStructuredText:
.. image:: https://joss.theoj.org/papers/10.21105/joss.01850/status.svg
   :target: https://doi.org/10.21105/joss.01850
```
This is how it will look in your documentation:

[![DOI](https://joss.theoj.org/papers/10.21105/joss.01850/status.svg)](https://doi.org/10.21105/joss.01850)

**We need your help!**

Journal of Open Source Software is a community-run journal and relies upon volunteer effort. If you'd like to support us please consider doing either one (or both) of the the following:

- Volunteering to review for us sometime in the future. You can add your name to the reviewer list here: https://joss.theoj.org/reviewer-signup.html
- Making a small donation to support our running costs here: https://numfocus.org/donate-to-joss
